docs/features/os.rst

```diff
@@ -149,7 +149,7 @@ single or double quotes)::
     (Cmd)
 
 Automating cmd2 apps from other CLI/CLU tools
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Is there a reason you are extending past the text above? (applies to next file below too)